pkg/controller/validate/validate.go

```diff
@@ -13,6 +18,35 @@ import (
 	mdbv1 "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1"
 )
 
+/*
+	{
+	  "type": "service_account",
+	  "project_id": "PROJECT_ID",
+	  "private_key_id": "KEY_ID",
+	  "private_key": "-----BEGIN PRIVATE KEY-----\nPRIVATE_KEY\n-----END PRIVATE KEY-----\n",
+	  "client_email": "SERVICE_ACCOUNT_EMAIL",
+	  "client_id": "CLIENT_ID",
+	  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+	  "token_uri": "https://accounts.google.com/o/oauth2/token",
+	  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+	  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/SERVICE_ACCOUNT_EMAIL"
+	  "universe_domain": "googleapis.com"
+	}
+*/
+type googleServiceAccountKey struct {
```

Comment:
If we do have that dependency I will check and use if it exists. If we do not have it, I prefer not to bring a new dependency on just for a single struct.

Reply:
I believe the package that would have that would be: `google.golang.org/api/iam/v1`
https://pkg.go.dev/google.golang.org/api/iam/v1#ServiceAccountKey

I do not see us already using it so, rather than importing it just for this I would rather copy the struct or the parts of it that would make sense.